libs/blocks/merch/merch.css

```diff
@@ -7,7 +7,7 @@ span[data-wcs-osi] {
   display: inline-block;
 }
 
-.price-strikethrough {
+span.placeholder-resolved[data-template="priceStrikethrough"] {
```

Comment:
at https://main--milo--adobecom.hlx.page/drafts/ilyas/merch

the markup looks is as following, and the strikethrough works for me on Safari.

```
<span is="inline-price" data-wcs-osi="A1xn6EL4pK93bWjM8flffQpfEL-bnvtoQKQAvkx574M" data-template="strikethrough"
    class="placeholder-resolved"><span class="price price-strikethrough"
        aria-label="Regularly at US$20.99 per month"><span class="price-currency-symbol">US$
            ...
        </span>
    </span>
</span>
```

Reply:
@Axelcureno can you double check please whether this is a valid issue?

Reply:
I have validated the issue is fixed after syncing my branch with latest main. Closing this PR.

Reply:
I tested again in CC, here are my findings:
-CTA button issue: Fixed.
-Strikethrough prices issue: Still happening, the second span only has 'price' class, missing 'strikethrough' class hence is not applying the text style. With this PR the issue would be gone since it uses data-template which is consistent.